foundation.go

```diff
@@ -155,12 +176,22 @@ func InitConsoleLogging(appgroup, app, version, branch, revision, buildDate stri
 
 // LogStartupMessage logs a default startup message for any Estafette application
 func LogStartupMessage(appgroup, app, version, branch, revision, buildDate string) {
+	startupProps := struct {
+		Branch    string `json:"branch"`
+		Revision  string `json:"revision"`
+		BuildDate string `json:"buildDate"`
+		GoVersion string `json:"goVersion"`
+		Os        string `json:"os"`
+	}{
+		branch,
+		revision,
+		buildDate,
+		goVersion,
+		runtime.GOOS,
+	}
+
 	log.Info().
-		Str("branch", branch).
-		Str("revision", revision).
-		Str("buildDate", buildDate).
-		Str("goVersion", goVersion).
-		Str("os", runtime.GOOS).
+		Interface("payload", startupProps).
```

Comment:
Please make it only do this for the `v3` format, not `console`, `json` and `stackdriver`.